### DISCUSSION TITLE
I created detailed documentation similar to Fruition's and a Showcase for y'all to show off your sites!

### DISCUSSION THREAD

**Discussion by DudeThatsErin** [velsa/notehost#26] [Show and tell]
https://dudethatserin.notion.site/NoteHost-982d31fcc8dd4799a18efcb074b0e63c

Simpler link:
https://techbyerin.com/notehost

Check it out! I hope it helps! I sent a PR (that should be merged soon) to update the readme as well.

**Comment by invictivi** [comment]
hey @DudeThatsErin that's amazing! 
I have been going through the instructions since yesterday ;)

I figured there was one step missing between 6 and 7
-> The actual Notehost installation

> Go into your working directory and run:
> 
> `npx notehost init <domain>`

It has to be after step 6. Step 11 is way too far in the process.
If I start from scratch, I don't have my domain catalog with files to edit, as mentioned in steps 7-10.

**Reply by DudeThatsErin** [reply]
Alright, updated on Notion, I'll submit a PR with it updated on the readme. :)

**Comment by slau8** [comment]
This is awesome; thank you for the instructions!

**Comment by DudeThatsErin** [comment]
@velsa Can you get this added to the Readme as a link so that people can easily find it?

**Reply by verbevent** [reply]
yep it was hard to find it haha

**Comment by verbevent** [comment]
Awesome work, thank you
